lib/board.rb

```diff
@@ -0,0 +1,53 @@
+class Board
+
+  attr_accessor :cells
+
+  def initialize
+    reset!
+  end
+
+  def reset!
+    @cells = [" ", " ", " ", " ", " ", " ", " ", " ", " "]
+  end
+
+  def display
+    puts " #{cells[0]} | #{cells[1]} | #{cells[2]} "
+    puts "-----------"
+    puts " #{cells[3]} | #{cells[4]} | #{cells[5]} "
+    puts "-----------"
+    puts " #{cells[6]} | #{cells[7]} | #{cells[8]} "
+  end
+
+  def position(input)
+    @cells[input.to_i - 1]
+  end
+
+  def full?
+    !@cells.include?(" ") ? true : false
+  end
+
+  def turn_count
+    @cells.count{|cell| cell != " "}
+  end
+
+  def taken?(input)
+    if position(input) == "X" || position(input) == "O"
```

Comment:
a conditional statement will return true or false implicitly by default. This removes the need for a if statement with explicit returns.